src/vivarium_data_requests/jun_17_laura_lamberti/model_specs/template.yaml

```diff
@@ -0,0 +1,15 @@
+plugins:
+    optional:
+        data:
+            controller: "vivarium_public_health.dataset_manager.ArtifactManager"
```

Comment:
Shouldn't this be the builder?

Reply:
well build_artifact was swapping that in for me but changing this will make it more explicit that these aren't really meant to be run, I like that